src/module.d.ts

```diff
@@ -400,7 +400,7 @@ export namespace plugins {
     var PositionPlugin : (settings: PositionSettings) => GriddlePlugin;
 }
 
-export const ColumnDefinition;
-export const RowDefinition;
+export const ColumnDefinition : any;
+export const RowDefinition : any;
```

Comment:
Failed to compile.

Error in node_modules/griddle-react/dist/module/module.d.ts
(403,14): error TS7005: Variable 'ColumnDefinition' implicitly has an 'any' type.

Error in node_modules/griddle-react/dist/module/module.d.ts
(404,14): error TS7005: Variable 'RowDefinition' implicitly has an 'any' type.